storybook/config.js

```diff
@@ -6,11 +6,132 @@ import i18n from '@/i18n'
 
 import vocabularyTheme from './theme'
 
+// Configuration of addon-viewport
+
+const newViewports = {
+
+  // Mobile devices
+
+  iPhoneX: {
+    name: 'iPhoneX',
+    styles: {
+      width: '375px',
+      height: '812px'
+    }
+  },
+  GooglePixel3: {
+    name: 'Google Pixel 3',
+    styles: {
+      width: '412px',
+      height: '824px'
+    }
+  },
+  Note9: {
+    name: 'Samsung Galaxy Note 9',
+    styles: {
+      width: '360px',
+      height: '740px'
+    }
+  },
+
+  // Tablet - Portrait
+
+  iPadPro: {
+    name: 'Apple iPad Pro',
+    styles: {
+      width: '1024px',
+      height: '1366px'
+    }
+  },
+  iPadAir: {
+    name: 'Apple iPad Air',
+    styles: {
+      width: '834px',
+      height: '1112px'
+    }
+  },
+  iPadMini: {
+    name: 'Apple iPad Mini',
+    styles: {
+      width: '768px',
+      height: '1024px'
+    }
+  },
+  iPad: {
+    name: 'Apple iPad 10.2 inch',
+    styles: {
+      width: '810px',
+      height: '1080px'
+    }
+  },
+
+  // Tablet - Landscape
```

Comment:
You don't need to redefine tablets in landscape actually. The viewport addon already has support for orientation change.